cmd/issuer-rest/startcmd/start.go

```diff
@@ -826,3 +846,19 @@ func getOauth2Config(authURL, tokenURL, redirectURL, clientID, secret string) *o
 
 	return config
 }
+
+func readProfiles(path string) ([]*operation.Profile, error) {
+	f, err := os.Open(path) //nolint:gosec
+	if err != nil {
+		return nil, fmt.Errorf("open file with profiles: %w", err)
+	}
+	defer f.Close() //nolint:gosec,errcheck
```

Comment:
Don't ignore the error return here. To handle the error, you can do something like this:

```
defer func() {
   err = f.Close()
    if err != nil {
       panic(fmt.Errorf("failed to close file: %w", err))
   }   
}()
```